phoenix-core/src/main/java/org/apache/phoenix/coprocessor/generated/ServerCachingProtos.java

```diff
@@ -5759,6 +5850,80 @@ public Builder clearIndexDataColumnCount() {
         return this;
       }
 
+      // optional string parentTableType = 24;
+      private java.lang.Object parentTableType_ = "";
+      /**
+       * <code>optional string parentTableType = 24;</code>
+       */
+      public boolean hasParentTableType() {
+        return ((bitField0_ & 0x00800000) == 0x00800000);
+      }
+      /**
+       * <code>optional string parentTableType = 24;</code>
+       */
+      public java.lang.String getParentTableType() {
+        java.lang.Object ref = parentTableType_;
+        if (!(ref instanceof java.lang.String)) {
+          java.lang.String s = ((com.google.protobuf.ByteString) ref)
+              .toStringUtf8();
+          parentTableType_ = s;
+          return s;
+        } else {
+          return (java.lang.String) ref;
+        }
+      }
+      /**
+       * <code>optional string parentTableType = 24;</code>
+       */
+      public com.google.protobuf.ByteString
+          getParentTableTypeBytes() {
```

Comment:
this can fit in one line, any reason to break to multiple? applies to other places as well

Reply:
This is auto generated file